### ISSUE TITLE
unarchive problem in download.py

### ISSUE THREAD

**Issue by jtang10** [cwpearson/graph-datasets#4]
When I ran the download.py script, I encountere the following issue:
```
Traceback (most recent call last):
  File "download.py", line 136, in <module>
    shutil.copyfileobj(f_in, f_out)
  File "/home/user/anaconda3/lib/python3.7/shutil.py", line 82, in copyfileobj
    fdst.write(buf)
TypeError: write() argument must be str, not bytes
```

The problem actually happens in line 135 and the fix should be 
```
with gzip.open(dst, 'rb') as f_in, open(extracted_path, 'wb') as f_out:
```
where the f_out should should have write access and in **binary** as well.

**Comment by cwpearson** [comment]
Can you give me the output of `python --version` for the python interpreter you used? Thanks!

**Comment by jtang10** [comment]
I'm on 3.7.1

**Comment by cwpearson** [comment]
I believe this is fixed in a71a50ac2c7213b393f2b342756afa9b8141cae2
